projects/core/styles/components/link.less

```diff
@@ -25,7 +25,6 @@
     color: inherit;
     text-decoration: none dashed currentColor;
     text-underline-offset: 0.2em;
-    text-decoration-thickness: 0.7px;
```

Comment:
Please put it back, it helps dashed line have smaller dash length which better resemble design.

Reply:
fixed